src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -143,7 +145,11 @@ class DialogmoteService(
                     varselUuid = motedeltakerArbeidstakerVarselIdPair.second,
                 )
 
```

Comment:
IntelliJ hinter om at koden for å lage motedeltakerArbeidstakerVarselIdPair og sende arbeidstakerVarsel og naermesteLederVarsel er duplisert på linje 214 osv. Det er vel også noe lignende varsel-kode lenger ned så kanskje man kunne refaktorert varsel-laging og utsending til en funksjon?

Reply:
Fant også at man kan destrukturere i kotlin (ref. https://www.baeldung.com/kotlin/destructuring-declarations), så vi kan f.eks. hente ut verdien vi er interessert i fra motedeltakerArbeidstakerVarselIdPair direkte (ala val `(id, uuid) = connection.createMotedeltakerVarselArbeidstaker`)  - blir kanskje litt mer lesbart enn å operere med `pair.second` osv?

Reply:
Gode forslag. Skal forsøke en refaktorering med utgangspunkt i dette.